### ISSUE TITLE
Value of type 'MICountryPicker' has no member 'delegate'

### ISSUE THREAD

**Issue by TheEngineerz** [mustafaibrahim989/MICountryPicker#3]
Heres my class starting

```
class SignUpController: UIViewController,MICountryPickerDelegate {
....
}
```

but i am getting this

![screen shot 2016-05-25 at 4 57 41 pm](https://cloud.githubusercontent.com/assets/5149947/15539094/d725f4c4-2299-11e6-9982-39daad2b7f98.png)

**Comment by mustafaibrahim989** [comment]
Ohh, It was an access modifiers issue. It's now fixed and available in 0.1.6.

**Comment by TheEngineerz** [comment]
Thanks Ibrahim
